libavformat/hls.c

```diff
@@ -1634,6 +1634,12 @@ static int open_segment(struct playlist *v)
                 if (ret != AVERROR_EXIT)
                     av_log(v->parent, AV_LOG_WARNING, "Failed to reload playlist %d\n",
                            v->index);
+                if (ret == AVERROR_HTTP_NOT_FOUND) {
```

Comment:
Go to the above reload when failed to get segment file
```
2024-12-05 18:04:48.832 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x5ac8a7c6efc0] request: GET /profile_0/media_segment_322.ts HTTP/1.1^M
User-Agent: IMAX StreamAware On-Air/nostamp-nostamp^M
Accept: */*^M
Range: bytes=0-^M
Connection: keep-alive^M
Host: 10.119.117.204:9999^M
Icy-MetaData: 1^M
^M

2024-12-05 18:04:48.832 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x5ac8a7c6efc0] HTTP error 404 Not Found
2024-12-05 18:04:48.832 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x5ac8a72f8000] Segment http://10.119.117.204:9999/profile_0/media_segment_322.ts, size -1, initial download failed
2024-12-05 18:04:48.832 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x5ac8a72f8000] Failed to open segment 322 of playlist 0

```

Reply:
The fix in this MR is for media playlist, the log is like:
`2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x55bc82f13000] Opening 'http://10.119.117.204:9999/profile_0/profile.m3u8' for reading
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x55bc84701200] request: GET /profile_0/profile.m3u8 HTTP/1.1^M
User-Agent: IMAX StreamAware On-Air/nostamp-nostamp^M
Accept: */*^M
Range: bytes=0-^M
Connection: keep-alive^M
Host: 10.119.117.204:9999^M
Icy-MetaData: 1^M
^M

2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x55bc84701200] HTTP error 404 Not Found
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [AVIOContext @ 0x55bc82aecdc0] Statistics: 540 bytes read, 0 seeks
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x55bc82f0e000] keepalive request failed for 'http://10.119.117.204:9999/profile_0/profile.m3u8' with error: 'Server returned 404 Not Found' when parsing playlist
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x55bc82f0e000] Opening 'http://10.119.117.204:9999/profile_0/profile.m3u8' for reading
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Original list of addresses:
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Address 10.119.117.204 port 9999
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Interleaved list of addresses:
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Address 10.119.117.204 port 9999
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Starting connection attempt to 10.119.117.204 port 9999
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [tcp @ 0x55bc8b4a3b00] Successfully connected to 10.119.117.204 port 9999
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x55bc83778540] request: GET /profile_0/profile.m3u8 HTTP/1.1^M
User-Agent: IMAX StreamAware On-Air/nostamp-nostamp^M
Accept: */*^M
Range: bytes=0-^M
Connection: keep-alive^M
Host: 10.119.117.204:9999^M
Icy-MetaData: 1^M
^M

2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x55bc83778540] HTTP error 404 Not Found
2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x55bc82f0e000] Failed to reload playlist 0
**2024-12-05 18:10:16.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [hls @ 0x55bc82f0e000] Going to reload playlist 0, reload count 2**
`

Reply:
For dash, when manifest.mpd  was deleted:
`2024-12-05 18:27:34.652 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x5abe694b4c00] request: GET /manifest.mpd HTTP/1.1^M
User-Agent: IMAX StreamAware On-Air/nostamp-nostamp^M
Accept: */*^M
Connection: keep-alive^M
Host: 10.119.117.204:9999^M
^M

2024-12-05 18:27:34.655 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x5abe694b4c00] HTTP error 404 Not Found
2024-12-05 18:27:34.655 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x5abe698da000] new fragment: cur[23] min[18] max[22] (prev_max[22])
2024-12-05 18:27:34.655 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x5abe698da000] new fragment outside playlist availabilit
2024-12-05 18:27:34.655 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x5abe694b6900] Segment not ready (reload_count: 4), retrying again in 2500ms`

Reply:
Dash: when got 404 from getting segment, keep retrying seems without any delay, finally exit after failures.
```

2024-12-05 18:53:59.919 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/init-stream0.m4s, size 865, initial download successful
2024-12-05 18:53:59.922 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00008.m4s, size 7451361, initial download successful
2024-12-05 18:53:59.923 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/init-stream1.m4s, size 726, initial download successful
2024-12-05 18:53:59.924 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00008.m4s, size 120616, initial download successful
2024-12-05 18:54:00.579 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00009.m4s, size 6271238, initial download successful
2024-12-05 18:54:00.583 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00009.m4s, size 120616, initial download successful
2024-12-05 18:54:00.597 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00010.m4s, size 5047507, initial download successful
2024-12-05 18:54:00.598 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00010.m4s, size 121388, initial download successful
2024-12-05 18:54:00.610 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00011.m4s, size 6258072, initial download successful
2024-12-05 18:54:00.611 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00011.m4s, size 120616, initial download successful
2024-12-05 18:54:00.626 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00012.m4s, size 7082370, initial download successful
2024-12-05 18:54:00.629 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00012.m4s, size 121388, initial download successful
2024-12-05 18:54:03.142 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00013.m4s, size 5433556, initial download successful
2024-12-05 18:54:03.144 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00013.m4s, size 120616, initial download successful
2024-12-05 18:54:08.158 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00014.m4s, size 6257967, initial download successful
2024-12-05 18:54:08.159 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00014.m4s, size 120616, initial download successful
2024-12-05 18:54:13.169 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00015.m4s, size 6379448, initial download successful
2024-12-05 18:54:13.170 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00015.m4s, size 121388, initial download successful
2024-12-05 18:54:18.184 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00016.m4s, size 6136460, initial download successful
2024-12-05 18:54:18.186 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00016.m4s, size 120616, initial download successful
2024-12-05 18:54:23.195 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00017.m4s, size 6258048, initial download successful
2024-12-05 18:54:23.196 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream1-00017.m4s, size 121388, initial download successful
2024-12-05 18:54:28.205 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.207 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.208 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.208 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.209 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.209 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.210 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.210 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.213 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.213 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.213 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.213 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.214 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.214 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.215 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.215 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.216 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.217 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.217 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.218 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.218 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.219 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.219 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.220 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.220 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.220 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.220 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.221 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.221 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.222 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.222 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.223 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.223 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.226 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.226 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.227 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.227 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.228 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.228 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.228 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.229 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.230 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.230 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.231 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.231 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.231 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.231 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.235 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.235 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.236 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.236 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.236 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.236 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.237 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.237 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.238 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.238 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.239 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.239 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.240 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.240 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.241 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.241 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.241 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.242 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.242 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.242 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.243 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.243 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.244 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.244 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.245 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.245 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.246 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.246 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.247 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.247 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.248 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.248 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.249 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.249 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.250 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.250 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.250 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.250 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.251 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.251 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.252 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.252 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.253 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.253 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.254 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.254 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.255 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.255 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.258 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.258 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.259 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.259 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.260 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.260 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.261 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.261 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.264 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.264 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.265 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.265 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.265 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.265 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.266 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.266 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.267 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.267 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.268 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.268 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.269 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.269 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.270 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.270 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.273 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.273 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.274 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.274 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.275 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.275 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.276 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.276 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.276 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.276 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.279 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.279 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.280 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.280 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.281 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.281 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.282 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.282 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.283 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.283 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.284 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.284 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.285 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.285 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.286 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.286 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.287 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.287 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.288 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.288 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.289 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.290 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.291 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.291 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.293 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.293 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.294 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.294 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.295 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.295 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.296 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.296 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.297 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.297 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.298 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.298 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.299 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.299 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.300 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.300 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.301 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.301 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.302 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.302 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.303 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.303 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.304 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.304 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.305 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.305 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.305 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.306 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.306 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.307 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.307 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.307 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.308 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.308 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.309 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6780] HTTP error 404 Not Found
2024-12-05 18:54:28.309 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.311 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3ad6900] HTTP error 404 Not Found
2024-12-05 18:54:28.311 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed
2024-12-05 18:54:28.312 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [http @ 0x6336d3496cc0] HTTP error 404 Not Found
2024-12-05 18:54:28.312 [analyzer_ffmpeg] MediaServer.cpp:109(ffmpegLogCB): [dash @ 0x6336d38bc000] Segment http://10.119.117.204:9999/chunk-stream0-00018.m4s, size -1, initial download failed

```

Reply:
We don't have 404 response code related integration tests

Reply:
If we receive a 500/300 for an HLS media playlist, we'll still continue to exit, correct?

IMO, we should add retries for all HTTP errors when fetching the media playlist and then wrap it with a feature flag to have the ability to turn that behaviour on or off.

As for the DASH issues where it retries forever when it can't get a segment - that's a known issue.
There are a whole bunch JIRA tickets that reference this behaviour (see SA-13466)

Reply:
If we receive a 500/300 for an HLS media playlist, we'll still continue to exit, correct?
--From the code, should exit.
I was checking how to test 500/300 response code for media playlist, but I haven't found a way to do it
That "chaos-stream-proxy" only can set error response code for segment.  
```
`
IMO, we should add retries for all HTTP errors when fetching the media playlist and then wrap it with a feature flag to have the ability to turn that behaviour on or off.`
```
I like this idea.